Example/OfflineViewController.swift

```diff
@@ -0,0 +1,90 @@
+import UIKit
+import Mapbox
+import MapboxDirections
+import MapboxCoreNavigation
+
+class OfflineViewController: UIViewController {
+    
+    var mapView: MGLMapView!
+    var resizableView: ResizableView!
+    var backgroundLayer = CAShapeLayer()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        mapView = MGLMapView(frame: view.bounds)
+        view.addSubview(mapView)
+        
+        backgroundLayer.frame = view.bounds
+        backgroundLayer.fillColor = #colorLiteral(red: 0.1450980392, green: 0.2588235294, blue: 0.3725490196, alpha: 0.196852993).cgColor
+        view.layer.addSublayer(backgroundLayer)
+        
+        resizableView = ResizableView(frame: CGRect(origin: view.center, size: CGSize(width: 50, height: 50)),
+                                      backgroundLayer: backgroundLayer)
+        
+        view.addSubview(resizableView)
+        
+        navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Download", style: .done, target: self, action: #selector(downloadRegion))
```

Comment:
Looks like we don’t have a Localizable.strings file for the example target yet, only for the framework targets. I didn’t originally expect that we would even translate the example application at all, but we wound up making the storyboard localizable, so we might as well make the rest of it localizable too.